webservices/server-integration/src/main/java/org/jboss/as/webservices/util/SubjectUtil.java

```diff
@@ -121,6 +129,54 @@ private static void addPrivateCredential(final Subject subject, final Object cre
         }
     }
 
+    public static SecurityIdentity convertToSecurityIdentity(Subject subject, Principal principal, SecurityDomain domain,
```

Comment:
Maybe just `toSecurityIdentity` name would be more consistent with existing `fromSecurityIdentity` method.
Otherwise looks ok I think.

Reply:
Fixed